marlowe-playground-client/src/BottomPanel/State.purs

```diff
@@ -0,0 +1,21 @@
+module BottomPanel.State
+  ( handleAction
+  ) where
+
+import Prelude hiding (div)
+import BottomPanel.Types (Action(..), State, _panelView, _showBottomPanel)
+import Data.Lens (assign, set)
+import Halogen (HalogenM, modify_)
+import MainFrame.Types (ChildSlots)
+
+handleAction ::
+  forall m panel panelAction.
+  Action panel panelAction ->
```

Comment:
I feel like `panelAction` should be `action`. It the action type of the "parent"

Reply:
changed